### DISCUSSION TITLE
github.com/dlclark/regexp2

### DISCUSSION THREAD

**Discussion by addice1019** [avelino/awesome-go#4997] [Ideas]
This package supports the use of richer regular expressions, complements the official regular expression functionality flaws and optimizes performance issues

github.com/dlclark/regexp2

**Comment by avelino** [comment]
to propose new projects you need to open pull requests including them, remembering that there must be a pull request for each project

[I recommend reading](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md)
